src/test/java/org/cactoos/io/LoggingInputStreamTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017-2018 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.io;
+
+import java.io.ByteArrayInputStream;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.Test;
+
+/**
+ * Test case for {@link LoggingInputStream}.
+ *
+ * @since 0.39
+ * @checkstyle JavadocMethodCheck (500 lines)
+ */
+public final class LoggingInputStreamTest {
+
+    @Test
+    public void readEmptyStream() throws Exception {
```

Comment:
@ilyakharlamov In case of an exception, what's the value returned by `read()`? Add a test to it.

Reply:
@fabriciofx Not sure if I understand that. If there is an exception of the underlying reader, there will be no return value, there will be an exception. Is that what you mean? To check if an exception is thrown?

Reply:
@ilyakharlamov I'm sorry, you're completely right. The idea is to add tests in an exception has raised, but I think we already have tests to do it, right?

Reply:
@fabriciofx I've added a test that checks for an exception